src/main/java/com/varukha/onlinebookstore/service/shoppingCart/ShoppingCartService.java

```diff
@@ -0,0 +1,22 @@
+package com.varukha.onlinebookstore.service.shoppingcart;
+
+import com.varukha.onlinebookstore.dto.cartitem.CartItemDto;
+import com.varukha.onlinebookstore.dto.cartitem.CreateCartItemRequestDto;
+import com.varukha.onlinebookstore.dto.cartitem.UpdateCartItemRequestDto;
+import com.varukha.onlinebookstore.dto.shoppingcart.ShoppingCartDto;
+import com.varukha.onlinebookstore.model.CartItem;
+import com.varukha.onlinebookstore.model.User;
+
+public interface ShoppingCartService {
+    void create(User user);
```

Comment:
Why do you need it on top level?

Reply:
I use it in UserServiceImpl in order to create shopping cart during user registration.